repository/cowsay/tests/cowsay-config-map/02-assert.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+binaryData:
```

Comment:
I don't think we need to check the exact content of the config map. Its presence along with the `name: cowsayinstance.deploy.main.genfiles.genwww.indexhtml` should be enough.

Reply:
yep... totally makes sense... removed